spec/js/views/todoView_spec.js

```diff
@@ -2,55 +2,44 @@ define([
   'views/todo_view',
   'model/todo',
   'collections/todoList',
-  'views/todoList_view',
   'ejs'
-], function (TodoView, Todo, TodoList, TodoListView) {
+], function (TodoView, Todo, TodoList) {
   var todoView, subject, model;
 
-  beforeAll(function () {
+  beforeEach(function () {
     model = new Todo({name: 'test', done: false});
+
     todoView = new TodoView({model: model});
+    todoView.render();
```

Comment:
Please do the `render` setup only for specs that require the view to be rendered. This limits you in your `render` specs -> you are not able to do proper setup.